go.mod

```diff
@@ -432,3 +449,7 @@ exclude (
 	go.etcd.io/etcd v0.0.0-20191023171146-3cf2f69b5738
 	go.etcd.io/etcd v0.5.0-alpha.5.0.20200910180754-dd1b699fc489
 )
+
+// `go mod tidy` pulls in 0.19.0 from somewhere, but apiextensions-apiserver and apiserver both want this version.
+// The latter breaks in hiveadmission otherwise.
+replace github.com/google/cel-go => github.com/google/cel-go v0.17.7
```

Comment:
Not exactly. If I upgrade cel-go to 19, hiveadmission breaks, but it's because hiveadmission => calls a thing in apiextensions-apiserver => calls a thing in cel-go that relies on cel-go being old. To upgrade cel-go, we would need a newer apiextensions-apiserver that has been updated to use the newer cel-go.

When I ran into the problem, I simply went and looked at what version of cel-go is required in apiextensions-apiserver's go.mod, and for [0.29.1](https://github.com/kubernetes/apiextensions-apiserver/blob/v0.29.1/go.mod#L11) (and in fact all the way up to [current master](https://github.com/kubernetes/apiextensions-apiserver/blob/efe8780f3d9b957e2efe4c5733e47793847c13ff/go.mod#L11)) -- it's this one.